x-pack/plugins/security_solution/public/alerts/components/alerts_table/index.tsx

```diff
@@ -53,9 +53,8 @@ import {
   displayErrorToast,
 } from '../../../common/components/toasters';
 
-export const ALERTS_TABLE_TIMELINE_ID = 'alerts-table';
-
 interface OwnProps {
+  timelineId: string;
```

Comment:
I am wondering if it will be safer to create a specific type for timeline id, for now, to avoid weird bug

Reply:
It sounds reasonable. I think the declaration of this type should belong to `timeline` and not to `alerts_table`. What do you think?